bedrock/base/templates/base-resp.html

```diff
@@ -60,6 +56,7 @@
     {# Global styles, hidden from IE7 and lower #}
     {% block site_css %}
       {% stylesheet 'responsive' %}
+      {% stylesheet 'tabzilla-static' %}
```

Comment:
I haven't dug into this PR yet, but given that the `tabzilla-static` CSS file is pretty tiny - is it actually a performance hit to include it like this and incur an extra http request, as opposed to bundling it in the `responsive` common CSS bundle?